src/xtgeo/grid3d/_grid_import.py

```diff
@@ -49,6 +50,23 @@ def from_file(
         result.update(_grid_import_xtgcpgeom.import_xtgcpgeom(gfile, **kwargs))
     elif fformat == FileFormat.HDF:
         result.update(_grid_import_xtgcpgeom.import_hdf5_cpgeom(gfile, **kwargs))
+    else:
+        extensions = FileFormat.extensions_string(
+            [
+                FileFormat.ROFF_BINARY,
+                FileFormat.ROFF_ASCII,
+                FileFormat.EGRID,
+                FileFormat.FEGRID,
+                FileFormat.GRDECL,
+                FileFormat.BGRDECL,
+                FileFormat.XTG,
+                FileFormat.HDF,
+            ]
```

Comment:
Ideally in #1164 this list becomes a property of the different class types, or something like that.